feat(wallet) extract and use transaction status for activity filtering

stefandunca opened the pull request:
### Update status-desktop [#10746](https://github.com/status-im/status-desktop/issues/10746)

- [x] extract and migrate existing status from receipt to the activity [#3540](https://github.com/status-im/status-go/pull/3540)
- [x] extend filter to include status extraction
- [x] extend filter to aggregate data from complex activity entries
- [x] expose status in API

ghost commented:
<!--prchecklist--> 
### Pull Request Checklist
- [ ] Have you updated the documentation, if impacted (e.g. [docs.status.im](https://status.im/docs/))?
- [ ] Have you tested changes with mobile?
- [ ] Have you tested changes with desktop?

status-im-auto commented:
### Jenkins Builds
<details>
<summary>Click to see older builds (73)</summary>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | b391dd4a | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3512/1/) | 2023-05-21 11:10:59 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230521-110744-b391dd-pr3512.zip) |
| :heavy_check_mark: | b391dd4a | [#1](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3512/1/) | 2023-05-21 11:12:37 | ~4 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230521-110744-b391dd-pr3512.aar) |
| :heavy_check_mark: | b391dd4a | [#1](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3512/1/) | 2023-05-21 11:20:48 | ~13 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3512/1//consoleText) |
| :heavy_check_mark: | b391dd4a | [#1](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3512/1/) | 2023-05-21 11:21:55 | ~14 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230521-110744-b391dd-pr3512.zip) |
| | | | | | | |
| :heavy_check_mark: | 3ab1c294 | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3512/2/) | 2023-05-22 13:42:53 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230522-134100-3ab1c2-pr3512.zip) |
| :heavy_check_mark: | 3ab1c294 | [#2](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3512/2/) | 2023-05-22 13:43:27 | ~2 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3512/2//consoleText) |
| :heavy_check_mark: | 3ab1c294 | [#2](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3512/2/) | 2023-05-22 13:44:25 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230522-134055-3ab1c2-pr3512.aar) |
| :heavy_check_mark: | 3ab1c294 | [#2](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3512/2/) | 2023-05-22 13:44:50 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230522-134055-3ab1c2-pr3512.zip) |
| | | | | | | |
| :heavy_check_mark: | f094c71f | [#3](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3512/3/) | 2023-05-22 14:13:59 | ~5 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230522-140839-f094c7-pr3512.zip) |
| :heavy_check_mark: | f094c71f | [#3](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3512/3/) | 2023-05-22 14:17:01 | ~8 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230522-140835-f094c7-pr3512.aar) |
| :heavy_check_mark: | f094c71f | [#3](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3512/3/) | 2023-05-22 14:19:34 | ~10 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230522-140839-f094c7-pr3512.zip) |
| :heavy_multiplication_x: | f094c71f | [#3](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3512/3/) | 2023-05-22 14:35:46 | ~27 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3512/3//consoleText) |
| :heavy_check_mark: | f094c71f | [#4](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3512/4/) | 2023-05-23 12:50:44 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230523-124847-f094c7-pr3512.zip) |
| :heavy_check_mark: | f094c71f | [#4](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3512/4/) | 2023-05-23 12:52:06 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230523-124847-f094c7-pr3512.zip) |
| :heavy_check_mark: | f094c71f | [#4](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3512/4/) | 2023-05-23 12:52:26 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230523-124847-f094c7-pr3512.aar) |
| :heavy_check_mark: | f094c71f | [#4](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3512/4/) | 2023-05-23 13:01:37 | ~12 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3512/4//consoleText) |
| | | | | | | |
| :x: | b49bcb70 | [#5](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3512/5/) | 2023-05-23 16:38:47 | ~23 sec | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3512/5/consoleText) |
| :x: | b49bcb70 | [#5](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3512/5/) | 2023-05-23 16:39:12 | ~47 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3512/5/consoleText) |
| :heavy_multiplication_x: | b49bcb70 | [#5](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3512/5/) | 2023-05-23 16:39:21 | ~57 sec | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3512/5//consoleText) |
| :x: | b49bcb70 | [#5](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3512/5/) | 2023-05-23 16:39:28 | ~1 min | `linux` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3512/5/consoleText) |
| | | | | | | |
| :x: | e75ef54b | [#6](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3512/6/) | 2023-05-24 20:07:45 | ~43 sec | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3512/6/consoleText) |
| :x: | e75ef54b | [#6](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3512/6/) | 2023-05-24 20:07:49 | ~43 sec | `linux` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3512/6/consoleText) |
| :x: | e75ef54b | [#6](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3512/6/) | 2023-05-24 20:07:54 | ~53 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3512/6/consoleText) |
| :heavy_multiplication_x: | e75ef54b | [#6](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3512/6/) | 2023-05-24 20:09:19 | ~2 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3512/6//consoleText) |
| | | | | | | |
| :x: | 5cb3a8b6 | [#7](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3512/7/) | 2023-05-25 17:39:22 | ~1 min | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3512/7/consoleText) |
| :x: | 5cb3a8b6 | [#7](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3512/7/) | 2023-05-25 17:43:59 | ~5 min | `linux` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3512/7/consoleText) |
| :x: | 5cb3a8b6 | [#7](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3512/7/) | 2023-05-25 17:44:25 | ~6 min | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3512/7/consoleText) |
| :heavy_multiplication_x: | 5cb3a8b6 | [#7](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3512/7/) | 2023-05-25 17:44:30 | ~6 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3512/7//consoleText) |
| | | | | | | |
| :x: | 62a957eb | [#8](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3512/8/) | 2023-05-26 20:10:31 | ~42 sec | `linux` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3512/8/consoleText) |
| :x: | 62a957eb | [#8](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3512/8/) | 2023-05-26 20:10:39 | ~50 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3512/8/consoleText) |
| :x: | 62a957eb | [#8](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3512/8/) | 2023-05-26 20:11:07 | ~1 min | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3512/8/consoleText) |
| :heavy_multiplication_x: | 62a957eb | [#8](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3512/8/) | 2023-05-26 20:11:20 | ~1 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3512/8//consoleText) |
| | | | | | | |
| :heavy_multiplication_x: | 0c9f2acc | [#9](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3512/9/) | 2023-05-28 21:43:38 | ~1 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3512/9//consoleText) |
| :heavy_check_mark: | 0c9f2acc | [#9](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3512/9/) | 2023-05-28 21:44:28 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230528-214148-0c9f2a-pr3512.zip) |
| :heavy_check_mark: | 0c9f2acc | [#9](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3512/9/) | 2023-05-28 21:44:53 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230528-214148-0c9f2a-pr3512.zip) |
| :heavy_check_mark: | 0c9f2acc | [#9](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3512/9/) | 2023-05-28 21:46:00 | ~4 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230528-214143-0c9f2a-pr3512.aar) |
| | | | | | | |
| :heavy_check_mark: | 7e7ccc2b | [#10](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3512/10/) | 2023-05-30 18:52:59 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230530-185039-7e7ccc-pr3512.zip) |
| :heavy_check_mark: | 7e7ccc2b | [#10](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3512/10/) | 2023-05-30 18:53:23 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230530-185039-7e7ccc-pr3512.zip) |
| :heavy_check_mark: | 7e7ccc2b | [#10](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3512/10/) | 2023-05-30 18:54:21 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230530-185039-7e7ccc-pr3512.aar) |
| :heavy_multiplication_x: | 7e7ccc2b | [#10](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3512/10/) | 2023-05-30 19:01:52 | ~11 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3512/10//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 373ec287 | [#11](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3512/11/) | 2023-05-31 11:45:08 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230531-114250-373ec2-pr3512.zip) |
| :heavy_check_mark: | 373ec287 | [#11](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3512/11/) | 2023-05-31 11:46:17 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230531-114250-373ec2-pr3512.zip) |
| :heavy_check_mark: | 373ec287 | [#11](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3512/11/) | 2023-05-31 11:46:49 | ~4 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230531-114247-373ec2-pr3512.aar) |
| | | | | | | |
| :heavy_check_mark: | 6e9e27df | [#12](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3512/12/) | 2023-05-31 11:52:01 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230531-114945-6e9e27-pr3512.zip) |
| :heavy_check_mark: | 6e9e27df | [#12](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3512/12/) | 2023-05-31 11:52:09 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230531-114942-6e9e27-pr3512.zip) |
| :heavy_check_mark: | 6e9e27df | [#12](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3512/12/) | 2023-05-31 11:53:01 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230531-114942-6e9e27-pr3512.aar) |
| :heavy_multiplication_x: | 6e9e27df | [#12](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3512/12/) | 2023-05-31 12:16:33 | ~13 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3512/12//consoleText) |
| | | | | | | |
| :heavy_check_mark: | ba7989ae | [#13](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3512/13/) | 2023-06-06 11:10:17 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230606-110659-ba7989-pr3512.zip) |
| :heavy_check_mark: | ba7989ae | [#13](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3512/13/) | 2023-06-06 11:10:37 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230606-110659-ba7989-pr3512.zip) |
| :heavy_check_mark: | ba7989ae | [#13](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3512/13/) | 2023-06-06 11:11:41 | ~4 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230606-110659-ba7989-pr3512.aar) |
| :heavy_multiplication_x: | ba7989ae | [#13](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3512/13/) | 2023-06-06 11:31:43 | ~24 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3512/13//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 12d79736 | [#14](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3512/14/) | 2023-06-06 11:30:53 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230606-112829-12d797-pr3512.zip) |
| :heavy_check_mark: | 12d79736 | [#14](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3512/14/) | 2023-06-06 11:38:24 | ~9 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230606-112831-12d797-pr3512.zip) |
| :heavy_check_mark: | 12d79736 | [#14](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3512/14/) | 2023-06-06 11:40:26 | ~11 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230606-112829-12d797-pr3512.aar) |
| :heavy_multiplication_x: | 12d79736 | [#14](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3512/14/) | 2023-06-06 11:59:37 | ~27 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3512/14//consoleText) |
| :heavy_check_mark: | 12d79736 | [#15](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3512/15/) | 2023-06-06 15:21:01 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230606-151913-12d797-pr3512.zip) |
| :heavy_check_mark: | 12d79736 | [#15](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3512/15/) | 2023-06-06 15:21:20 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230606-151909-12d797-pr3512.zip) |
| :heavy_check_mark: | 12d79736 | [#15](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3512/15/) | 2023-06-06 15:23:28 | ~4 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230606-151909-12d797-pr3512.aar) |
| :heavy_multiplication_x: | 12d79736 | [#15](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3512/15/) | 2023-06-06 15:40:37 | ~21 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3512/15//consoleText) |
| :heavy_check_mark: | 12d79736 | [#16](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3512/16/) | 2023-06-08 11:57:05 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230608-115459-12d797-pr3512.zip) |
| :heavy_check_mark: | 12d79736 | [#16](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3512/16/) | 2023-06-08 11:57:15 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230608-115459-12d797-pr3512.zip) |
| :heavy_check_mark: | 12d79736 | [#16](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3512/16/) | 2023-06-08 11:59:24 | ~4 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230608-115459-12d797-pr3512.aar) |
| :heavy_check_mark: | 12d79736 | [#16](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3512/16/) | 2023-06-08 12:07:37 | ~12 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3512/16//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 09d1fe2c | [#17](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3512/17/) | 2023-06-08 12:11:43 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230608-120750-09d1fe-pr3512.zip) |
| :heavy_check_mark: | 09d1fe2c | [#17](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3512/17/) | 2023-06-08 12:17:34 | ~9 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230608-120750-09d1fe-pr3512.aar) |
| | | | | | | |
| :heavy_check_mark: | 59b4974d | [#18](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3512/18/) | 2023-06-08 12:16:32 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230608-121345-59b497-pr3512.zip) |
| :heavy_check_mark: | 59b4974d | [#18](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3512/18/) | 2023-06-08 12:25:24 | ~7 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230608-121739-59b497-pr3512.aar) |
| :heavy_check_mark: | 59b4974d | [#18](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3512/18/) | 2023-06-08 12:25:32 | ~10 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230608-121503-59b497-pr3512.zip) |
| :heavy_multiplication_x: | 59b4974d | [#18](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3512/18/) | 2023-06-08 12:43:12 | ~28 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3512/18//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 0a4f3dd7 | [#19](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3512/19/) | 2023-06-09 18:48:55 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230609-184736-0a4f3d-pr3512.zip) |
| :heavy_check_mark: | 0a4f3dd7 | [#19](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3512/19/) | 2023-06-09 18:49:19 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230609-184736-0a4f3d-pr3512.aar) |
| :heavy_check_mark: | 0a4f3dd7 | [#19](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3512/19/) | 2023-06-09 18:50:04 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230609-184736-0a4f3d-pr3512.zip) |
| :heavy_multiplication_x: | 0a4f3dd7 | [#19](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3512/19/) | 2023-06-09 19:10:25 | ~22 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3512/19//consoleText) |
</details>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | fbc43ffb | [#20](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3512/20/) | 2023-06-09 18:58:29 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230609-185721-fbc43f-pr3512.aar) |
| :heavy_check_mark: | fbc43ffb | [#20](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3512/20/) | 2023-06-09 18:59:37 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230609-185721-fbc43f-pr3512.zip) |
| :heavy_check_mark: | fbc43ffb | [#20](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3512/20/) | 2023-06-09 18:59:44 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230609-185721-fbc43f-pr3512.zip) |
| :heavy_multiplication_x: | fbc43ffb | [#20](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3512/20/) | 2023-06-09 19:33:48 | ~23 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3512/20//consoleText) |
| | | | | | | |
| :heavy_check_mark: | f9e23527 | [#21](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3512/21/) | 2023-06-09 19:54:19 | ~57 sec | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230609-195321-f9e235-pr3512.zip) |
| :heavy_check_mark: | f9e23527 | [#21](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3512/21/) | 2023-06-09 19:55:29 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230609-195321-f9e235-pr3512.zip) |
| :heavy_check_mark: | f9e23527 | [#21](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3512/21/) | 2023-06-09 19:58:01 | ~4 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230609-195321-f9e235-pr3512.aar) |
| :heavy_check_mark: | f9e23527 | [#21](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3512/21/) | 2023-06-09 20:04:47 | ~11 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3512/21//consoleText) |

stefandunca committed:
feat(wallet) filter activity by status

Main changes:

- extend DB query include status for transactions and pending_transactions
- extend DB query to aggregate data from complex activity entries
- extend tests to include the new status values

Other changes:

- Improve tests with mocked addresses in DB to have a true to/from state

Update status-desktop #10746

dlipicar commented on services/wallet/activity/activity.go:
```diff
@@ -198,6 +199,9 @@ const (
 	// UNION ALL is used to avoid the overhead of DISTINCT given that we don't expect to have duplicate entries outside
 	// the sender and receiver addresses being in the list which is handled separately
+	//
+	// Only status FailedAS, PendingAS and CompleteAS are returned. FinalizedAS is for bundled transactions
```
not sure if I'm mixing concepts, didn't the "Finalized" state mean the block for this transaction is [finalized](https://docs.symbol.dev/concepts/block.html#finalization)? In that case, it wouldn't matter if its a single transaction or a transaction bundle.

stefandunca replied:
You are right. I removed the bundle reference.

dlipicar reviewed (approved):
Just havethat question about what we call a "Finalized" activity. If the definition used here is the one we want, the changes look fine
